File: Pemrograman 1/MODUL 5/Bahasa Python/PRAK505-2410817110002-Akhmad_Daffa_Azzikri.py
def Biodata(tahun_lahir, nama, asal):
    tahun_sekarang = 2020
    print(f"Perkenalkan Nama Saya : {nama}")
    print(f"Umur Saya : {tahun_sekarang - tahun_lahir}")
    print(f"Saya Adalah Angkatan : {tahun_sekarang}")
    print(f"Asal Saya dari : {asal}")

tahun_lahir = int(input())
nama = input()
asal = input()

Biodata(tahun_lahir, nama, asal)

File: Pemrograman 1/MODUL 5/Bahasa C/PRAK505-2410817110002-Akhmad_Daffa_Azzikri.c
#include <stdio.h>

void Biodata(int tahunLahir, char nama [], char asal[]) {
    int tahun_sekarang = 2020;
    printf("Perkenalkan Nama Saya : %s\n", nama);
    printf("Umur Saya : %d\n", tahun_sekarang - tahunLahir);
    printf("Saya Adalah Angkatan : %d \n", tahun_sekarang);
    printf("Asal Saya dari : %s\n", asal);
}

int main() {
    int tahunLahir;
    char nama[20], asal[20];
    scanf("%d", &tahunLahir);
    scanf(" %[^\n]%*c", nama);
    scanf(" %[^\n]%*c", asal);
    Biodata(tahunLahir, nama, asal);
    return 0;
}
